app/src/main/java/org/sopt/sample/home/adapter/RepoAdapter.kt

```diff
@@ -0,0 +1,71 @@
+package org.sopt.sample.home.adapter
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.appcompat.content.res.AppCompatResources
+import androidx.recyclerview.widget.RecyclerView
+import org.sopt.sample.databinding.LayoutRvHeaderBinding
+import org.sopt.sample.databinding.LayoutGithubRepoBinding
+import org.sopt.sample.home.data.Repo
+
+class RepoAdapter(context: Context) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+    private val HEADER = 0
+    private val ITEM = 1
+    private val inflater by lazy { LayoutInflater.from(context) }
+    private var repoList: List<Repo> = emptyList()
+
+    override fun getItemViewType(position: Int): Int {
+        return if (position == 0) {
+            HEADER
+        } else ITEM
+    }
```

Comment:
오호!! 아주 깔끔하게 잘하셨군요~!! 최곱니다 ㅎㅎ 이렇게도 변경이 가능할 것 같아요! 상단에 각 뷰타입을 상수로 정의했으니!

```suggestion
    override fun getItemViewType(position: Int): Int {
        return when (position) {
            0 -> HEADER
        else -> ITEM
    }
    }
```